src/nrniv/CMakeLists.txt

```diff
@@ -334,6 +334,10 @@ if(NRN_HAVE_NVHPC_COMPILER)
     ${PROJECT_SOURCE_DIR}/src/oc/math.cpp
     PROPERTIES COMPILE_DEFINITIONS NVHPC_CHECK_FE_EXCEPTIONS=1 COMPILE_OPTIONS
                                                                "${NVHPC_MATH_COMPILE_OPTIONS}")
+  # Versions of nvc++ around ~22.5 up to at least 23.1 have problems with this file see
+  # https://github.com/BlueBrain/CoreNeuron/issues/888
+  set_source_files_properties(${PROJECT_SOURCE_DIR}/src/ivoc/ivocvect.cpp PROPERTIES COMPILE_OPTIONS
```

Comment:
I'm just wondering whether there is any big performance degradation by `-O1` since `IvocVect` is responsible for all the `h.Vector()` calls

Reply:
I still cheer for  NVHPC -> `-O1` everywhere :D 

There's a lot of logic in ivocvect.cpp and we may get performance degradation, however I find that if we start mitigating these specific things we will end up doing a lot of custom checks. We should discuss this today.